post/templates/post/update_post.html

```diff
@@ -1,6 +1,7 @@
 {% extends 'post/base.html' %}
 {% block title %}Update a Post{% endblock %}
 {% block seo %}
+    <meta property="og:title" content="Update a Post">
```

Comment:
content="Update a Post"  **/**>. 
do you not need that forward slash?

Reply:
it is a way of showing the end of the tag, but i can delete it to have everywhere the same